src/pages/2021/best-of-penn/index.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react'
+import { Helmet } from 'react-helmet'
+import { useStaticQuery, graphql } from 'gatsby'
+
+import { BackgroundWrapper } from '../../../components/2021/best-of-penn/shared'
+import TanComponent from '../../../components/2021/best-of-penn/TanComponent.js'
+import BlueComponent from '../../../components/2021/best-of-penn/BlueComponent.js'
+const IndexPage = () => {
+  const data = useStaticQuery(graphql`
```

Comment:
this looks a bit fugly, but run prettier and see what it makes of it, and then defer to that.